docs-markdown/.vscode/launch.json

```diff
@@ -30,6 +32,22 @@
                 "${workspaceFolder}/out/src/**/*.js"
             ],
             "preLaunchTask": "mocha"
+        },
+        {
+            "name": "Metadata Controller Extension Tests",
```

Comment:
Hi @lamebrain , I think we can remove this section because mocha will pick up every test in the **test** directory by default so the existing "extension tests" section should be enough.  

I removed the "Metadata Controller Extension Tests" block then ran the tests locally and everything worked fine.

Reply:
I added this because I didn't want to run all the tests when working on my changes. Might be useful/save time to break up the tests like this.

Reply:
Ah, makes sense.